shared/constants.py

```diff
@@ -1,9 +1,61 @@
 # Constants used throughout the bot
 # Ensure the servers you run the bot in contain the required channel names
-GENERAL_CHANNEL_ID = "🌺│home"
-GOVERNANCE_CHANNEL_ID = "improvement-props"
-GOVERNANCE_BUDGET_CHANNEL_ID = "budgetary-props"
-GOVERNANCE_TALK_CHANNEL_ID = "🐬│governance"
+GENERAL_CHANNEL = "🌺│home"
+GOVERNANCE_CHANNEL = "improvement-props"
+GOVERNANCE_BUDGET_CHANNEL = "budgetary-props"
+GOVERNANCE_TALK_CHANNEL = "🐬│governance"
+INTRODUCE_YOURSELF_CHANNEL = "🌊│introduce-yourself"
```

Comment:
having fallbacks and really good notifications and error handling for this will be very helpfull if anything breaks here in the future and to inform people

Reply:
Comments help here too

Reply:
No worries I will add to this

> having fallbacks and really good notifications and error handling for this will be very helpfull if anything breaks here in the future and to inform people